Tests/Fixtures/config/config.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:fm-elfinder="http://helios-ag.github.io/schema/dic/fm_elfinder"
+>
+    <fm-elfinder:config configuration-provider="app.configurator.custom">
+        <fm-elfinder:instance name="default"
+            locale="en"
+            editor="tinymce"
+            editor-template="Elfinder/editor.html.twig"
+            fullscreen="false"
+            include-assets="false"
+            tinymce-popup-path="/pop-up"
+            enable-user-integration="true"
+            relative-path="false"
+        >
+            <fm-elfinder:connector debug="true">
+                <fm-elfinder:root name="uploads"
+                    driver="LocalFileSystem"
+                    path="uploads"
+                    showhidden="true"
+                    alias="foo"
+                    treeDeep="1"
```

Comment:
XML conventions are to only use lowercase letters and dashes. Is there anything wrong with converting this to `tree_deep` (and `tree-deep` in XML)?